src/Reshaping.jl

```diff
@@ -32,19 +34,35 @@ function reshape_supercell(sys::System{N}, shape) where N
 end
 
 
-# Transfer homogeneous interactions from `sys.origin` to reshaped `sys`
-function set_interactions_from_origin!(sys::System{N}) where N
-    origin = sys.origin
-    new_ints  = interactions_homog(sys)
-    orig_ints = interactions_homog(origin)
+# Transfer homogeneous interactions from `orig` to reshaped `sys`.
```

Comment:
Should "homogenous" be removed from the description here?

Reply:
Updated